README.md

```diff
@@ -220,6 +240,9 @@ Some of them can also be used for input schema file.
 - SQLite3
   - Table: CREATE TABLE, DROP TABLE
   - View: CREATE VIEW, DROP VIEW
+- SQLServer
```

Comment:
```suggestion
- SQL Server
```

is correct. ref. https://www.microsoft.com/ja-jp/sql-server/sql-server-downloads